eva-accession-core/src/main/java/uk/ac/ebi/eva/accession/core/persistence/SubmittedVariantInactiveEntity.java

```diff
@@ -18,30 +18,48 @@
 package uk.ac.ebi.eva.accession.core.persistence;
 
 import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
 import uk.ac.ebi.ampt2d.commons.accession.persistence.mongodb.document.InactiveSubDocument;
 
 import uk.ac.ebi.eva.accession.core.ISubmittedVariant;
-
-import java.util.Objects;
+import uk.ac.ebi.eva.accession.core.utils.ISubmittedVariantComparator;
 
 @Document
 public class SubmittedVariantInactiveEntity extends InactiveSubDocument<Long> implements ISubmittedVariant {
 
+    @Field("asm")
     private String assemblyAccession;
 
+    @Field("tax")
     private int taxonomyAccession;
 
+    @Field("study")
     private String projectAccession;
 
     private String contig;
```

Comment:
I did this comment because I thought those fields were not being stored because they didn't have "@Field" annotation.  They are stored so this comment can be safely ignored.